.github/workflows/ci.yml

```diff
@@ -40,9 +40,10 @@ jobs:
     strategy:
       matrix:
         image:
-          - 'swift:5.6'
           - 'swift:5.7'
           - 'swift:5.8'
```

Comment:
Still planning on supporting these?

Reply:
No harm, doing that here. I've use this outside of server development so its minimum supported swift version isn't linked to the minimum version for HB